File: 卡码/KamaCoder/Go/test15.go
//15. 神秘字符
package main

import (
	"fmt"
)

func main(){
	var n int
	fmt.Scanf("%d\n",&n)
	for i:=0;i<n;i++ {
		var str1,str2 string
		fmt.Scanln(&str1)
		fmt.Scanln(&str2)
		middle:=len(str1)/2
		var ret=str1[:middle]+str2+str1[middle:]
		fmt.Println(ret)
	}
}

File: 卡码/KamaCoder/Python/test15.py
#15. 神秘字符
import sys

def merge(str1,str2):
	middle=len(str1)//2
	return str1[:middle]+str2+str1[middle:]

def main():
	n=int(input())
	for i in range(n):
		str1=input()
		str2=input()
		print(merge(str1,str2))

if __name__ == '__main__':
	main()
